<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>164</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="distinct">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>避免重复</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="head_search">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>头搜索</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="middle_search">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>中间搜索</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="exclude_search">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>反搜</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>423</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>查找学号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="StuNo3"/>
    </item>
    <item row="0" column="4">
     <widget class="QCheckBox" name="StuNo3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>查找电话</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="PhoneNumber3"/>
    </item>
    <item row="0" column="7">
     <widget class="QCheckBox" name="PhoneNumber3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>查找姓名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="Name3"/>
    </item>
    <item row="1" column="4">
     <widget class="QCheckBox" name="Name3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>查找奖励</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="Award3"/>
    </item>
    <item row="1" column="7">
     <widget class="QCheckBox" name="Award3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>查找年龄</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Age3"/>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="Age3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="Age32"/>
    </item>
    <item row="2" column="4">
     <widget class="QCheckBox" name="Age32_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>查找处分</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLineEdit" name="Punish3"/>
    </item>
    <item row="2" column="7">
     <widget class="QCheckBox" name="Punish3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>查找院系</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="College3"/>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="College3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>查找籍贯</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLineEdit" name="NativePlace3"/>
    </item>
    <item row="3" column="7">
     <widget class="QCheckBox" name="NativePlace3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>查找专业</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLineEdit" name="Major3"/>
    </item>
    <item row="4" column="4">
     <widget class="QCheckBox" name="Major3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>查找家长信息</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLineEdit" name="ParentsInfo3"/>
    </item>
    <item row="4" column="7">
     <widget class="QCheckBox" name="ParentsInfo3_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
